clang/include/clang/Driver/Options.td

```diff
@@ -2649,6 +2649,14 @@ def fsanitize_undefined_strip_path_components_EQ : Joined<["-"], "fsanitize-unde
   HelpText<"Strip (or keep only, if negative) a given number of path components "
            "when emitting check metadata.">,
   MarshallingInfoInt<CodeGenOpts<"EmitCheckPathComponentsToStrip">, "0", "int">;
+def fno_sanitize_top_hot_EQ
```

Comment:
It could be confusing that a -fno option doesn't have a positive form. Consider fsanitize-skip..

Reply:
SGTM - I like the wording! @vitalybuka WDYT?

Reply:
There are fno_inline, fno_common, fno_builtin which have no positive form.
So the current naming is LGTM.

Reply:
These options have positive forms: -finline/-finline-functions -fcommon -fbuiltin

Reply:
Ops, I didn't bother to search the rest of the file. Most options declare posetive/negative nearby.

Reply:
`-fsanitize-skip-hot=` then? `top` is the name feels redundant?

Reply:
How about `-fsanitize-skip-hotness` or `-fsanitize-skip-hot-cutoff`, to imply some continuum? (`-fsanitize-skip-hot=1` could seem like a binary setting.)

Reply:
"seem like a binary setting" applies to "top_hot" 

`-fsanitize-skip-hot-cutoff` is probably better

Reply:
Ok, I'll change it to `-fsanitize-skip-hot-cutoff`